util/design/secded_gen.py

```diff
@@ -235,11 +229,19 @@ def generate(cfgs, args, seed):
         suffix = CODE_OPTIONS[codetype]
         codes = []
 
-        # update value based on target selection
-        codes = globals()["{}_code".format(codetype)](cfg['k'], cfg['m'])
+        # The hsiao_code generator uses (pseudo)random values to pick good ECC
+        # constants. Rather than exposing the seed, we pick a fixed one here to
+        # ensure everything stays stable in future.
+        old_rnd_state = random.getstate()
+        random.seed(123)
```

Comment:
I think this constant needs to be hoisted to a constant rather than having it hidden in here.

Reply:
Meh, ok. I'll make it a private global.